packages/wonder-blocks-form/src/components/labeled-text-field.js

```diff
@@ -29,10 +64,19 @@ type State = {|
 
 export default class LabeledTextField extends React.Component<Props, State> {
     static defaultProps: DefaultProps = {
+        type: "text",
         disabled: false,
     };
 
+    constructor(props: Props) {
+        super(props);
+        if (props.initialValue) {
+            this.state.value = props.initialValue;
+        }
+    }
+
     state: State = {
+        value: "",
         error: null,
         focused: false,
     };
```

Comment:
Is it good to have it like this? Or would removing the "constructor" and changing the state class property to something like below be better?
```js
state: State = {
    value: this.state.value ? this.state.value : "",
    error: null,
    focused: false,
};
```

Reply:
Personally I prefer doing `this.state = { ... }` in the constructor.  I find it easier to reason about especially since not everyone is up-to-date on all the new-fangled JavaScript features.

Reply:
I think it's better if you centralize that in a single place, and also think that it would be better moving everything to the `constructor`.